code-challenges.test.js

```diff
@@ -28,42 +33,172 @@ const numbersArray2 = [24, 27, 30, 33, 36]
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const num1 = 15
-// Expected output: "15 is divisible by three"
-const num2 = 0
-// Expected output: "0 is divisible by three"
-const num3 = -7
-// Expected output: "-7 is not divisible by three"
+
+
+// describe("divisibleBy3", () => {
+//     it("decides if the number is evenly divisible by three or not", () => {
+//       expect(divisibleBy3(num)).toEqual("is divisisible by three")
+//       expect(!divisibleBy3(num)).toEqual("is not divisible by three")
+//     })
+// })
+
+// FAIL  week-2-assessment-VMMarin/code-challenges.test.js
+// divisibleBy3
+//   ✕ decides if the number is evenly divisible by three or not (1 ms)
+
+// ● divisibleBy3 › decides if the number is evenly divisible by three or not
+
+//   ReferenceError: divisibleBy3 is not defined
+
+//     33 | describe("divisibleBy3", () => {
+//     34 |     it("decides if the number is evenly divisible by three or not", () => {
+//   > 35 |       expect(divisibleBy3(num)).toEqual("is divisisible by three")
+//        |       ^
+//     36 |       expect(!divisibleBy3(num)).toEqual("is not divisible by three")
+//     37 |     })
+//     38 | })
+
+
+
+
+
+
+// const num1 = 15
+// // Expected output: "15 is divisible by three"
+// const num2 = 0
+// // Expected output: "0 is divisible by three"
+// const num3 = -7
+// // Expected output: "-7 is not divisible by three"
 
 
 // b) Create the function that makes the test pass.
 
+//PSEUDOCODE:
+//Input will be num variable
+//use modulo to determine if numbers are evenly divisible by three
+//if statement
+//return num is or is not divisible by three
+
+let divByThree = (array) => {
+return array.map(value => {
+    if(value % 3 === 0) {
+        return "Is divisible by three"
+    } else {
+        return "is not divisible by three"
+        }
+        })
+    }
+divByThree(num1)
+divByThree(num2)
+divByThree(num3)
+
 
 
 // --------------------2) Create a function that takes in an array of words and returns an array with all the words capitalized.
 
 // a) Create a test with expect statements for each of the variables provided.
 
+
+
+// describe("capitalizedArray", () => {
+//     it("returns an array with all the words capitalized", () => {
+//         expect(randomNouns1.map.toUpperCase).toEqual(["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"])
+//         expect(randomNouns2.map.toUpperCase).toEqual(["Temperature", "Database", "Chopsticks", "Mango", "Deduction"])
+
+//     })
+// })
+
+// const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
+// // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
+// const randomNouns2 = ["temperature", "database", "chopsticks", "mango", "deduction"]
+// // Expected output: ["Temperature", "Database", "Chopsticks", "Mango", "Deduction"]
+
```

Comment:
The goal of the test is that it is automating your function invocation. The only thing the test should do is call the function, pass an argument, and then expect an outcome.